main.go

```diff
@@ -14,6 +15,15 @@ import (
 )
 
 func main() {
+	delay_sec, err := strconv.Atoi(os.Getenv("GATUS_DELAY_START_SECONDS"))
+	if err != nil {
+		// If we can't get GATUS_DELAY_START_SECONDS from environment variable
+		// set delay_sec to 0
+		log.Println("Error when parsing the value of GATUS_DELAY_START_SECONDS")
+		delay_sec = 0
+	}
+	// Wait for delay_sec
+	time.Sleep(time.Duration(delay_sec) * time.Second)
```

Comment:
```suggestion
	if delayInSeconds, _ := strconv.Atoi(os.Getenv("GATUS_DELAY_START_SECONDS")); delayInSeconds > 0 {
		log.Printf("Delaying start by %d seconds", delayInSeconds)
		time.Sleep(time.Duration(delayInSeconds) * time.Second)
	}
```

Reply:
@TwiN 
I've made the requested changes, and add an example under .example/ directory
Please take a look when you have time, thank you!